core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -103,11 +103,17 @@
                 </div>
                 <div ng-if="!exploration.isInteractionInline(activeCard.stateName)"
                      style="opacity: 0.8;">
-                  <md-button class="instructions-button"
-                          ng-click="showSupplementalCardIfScreenIsNarrow()">
-                    <[exploration.getInteractionInstructions(activeCard.stateName)]>
-                    <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
-                  </md-button>
+		     <div ng-if ="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" 
+			     style="text-transform: uppercase;padding: 6px 12px;"> 
+				  <[exploration.getInteractionInstructions(activeCard.stateName)]>
+		     </div>
+		   <div ng-if="(!isCurrentSupplementalCardNonempty() || isScreenNarrowAndShowingTutorCard())">
+			<md-button class="instructions-button"
+			     ng-click="showSupplementalCardIfScreenIsNarrow()">
+				  <[exploration.getInteractionInstructions(activeCard.stateName)]>
+			          <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
```

Comment:
Read the original ticket incorrectly, and switched arrow placement in implementation. Now put arrow in proper place. Thanks Sourav.